src/ser/mod.rs

```diff
@@ -1117,48 +1115,7 @@ mod tests {
             serialized == r#"{"her_name":"aline","my_name":"joseph"}"#
                 || serialized == r#"{"my_name":"joseph","her_name":"aline"}"#
         );
-    }
-
-    #[test]
-    fn map_serialization_matches_json_serde() {
```

Comment:
Why remove this test?

Reply:
The idea is that the test pretty much does the same as the `assert_serde_json_serialize_eq!` implementation which is now embedded in `fn hash_map()` for various maps